src/hooks/useAddSearchParam.ts

```diff
@@ -11,17 +11,29 @@ export const useAddSearchParam = () => {
     ({
       name,
       value,
+      params,
       clean,
     }: {
-      name: string;
-      value: string;
+      name?: string;
+      value?: string;
+      params?: Record<string, string>;
       clean?: boolean;
     }) => {
-      const params = new URLSearchParams(
```

Comment:
I will revert this in the next review comment. We don't think we will use these changes from here on